debian/rules

```diff
@@ -191,6 +185,6 @@ override_dh_gencontrol:
 # white list file only starting from Debian Stretch and Ubuntu Xenial.
 # To find more, grep build logs for 'but is not installed to anywhere'.
 %:
-	dh $@ --parallel --with systemd --fail-missing
+	dh $@ --parallel
```

Comment:
Per https://manpages.debian.org/testing/debhelper/dh_missing.1.en.html

> Please note that in compat 11 and earlier without either of these options, dh_missing will silently do nothing. In compat 12, --list-missing is the default. In compat 13 and later, --fail-missing is the default.

From my understanding, this change will make it so that the build process will no longer fail on missing files. Is this intended and safe?

Reply:
I'll add it back as we are using compat 11 not compat 12 which is not building on Debian 10 proberly.